ITensorGPU/test/Project.toml

```diff
@@ -5,3 +5,7 @@ ITensors = "9136182c-28ba-11e9-034c-db9fb085ebd5"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```

Comment:
Do we need `Pkg` as a test dependency? I don't see it used anywhere.

Reply:
I don't think it is necessary